lib/routemaster/cache.rb

```diff
@@ -95,6 +95,29 @@ def bust(url)
     #
     # @return [Response], which responds to `status`, `headers`, and `body`.
     def get(url, version:nil, locale:nil)
+      response = with_caching(url, version: version, locale: locale) do
+        # fetch data
+        headers = {
+          'Accept' => version ?
+          "application/json;v=#{version}" :
+          "application/json"
+        }
+        headers['Accept-Language'] = locale if locale
+        @fetcher.get(url, headers: headers)
+      end
+      Response.new(response)
+    end
+
+    # Like {#get}, but schedules any request in the background using a thread
+    # pool. Handy to issue lots of requests in parallel.
+    #
+    # @return [FutureResponse], which responds to `status`, `headers`, and `body`
+    # like [Response].
+    def fget(*args)
+      FutureResponse.new { get(*args) }
+    end
+
+    def with_caching(url, version: nil, locale: nil)
```

Comment:
I'd suggest naming this `#fetch` out of similarity with the `ActiveSupport::Cache#fetch` interface.
Can you add a doc comment here, too?